### PR TITLE
Initial import

### PR DESCRIPTION
**Pull request by sagikazarmark**

### CODE REVIEW

File: src/ConsoleProgressWriter.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+     */
+    public function finish()
+    {
+        $this->progress->finish();
+    }
+
+    /**
+     * @return string
+     */
+    public function getVerbosity()
```
Comment (sagikazarmark) [inline src/ConsoleProgressWriter.php]: This getter should be removed

File: src/ConsoleProgressWriter.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    {
+        return $this->verbosity;
+    }
+
+    /**
+     * @return integer
+     */
+    public function getRedrawFrequency()
```
Comment (sagikazarmark) [inline src/ConsoleProgressWriter.php]: This getter should be removed

File: src/ConsoleTableWriter.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    /**
+     * @var array
+     */
+    private $firstItem;
+
+    /**
+     * @param OutputInterface $output
+     * @param Table           $table
+     */
+    public function __construct(OutputInterface $output, Table $table) {
```
Comment (sagikazarmark) [inline src/ConsoleTableWriter.php]: Output is not even used in the class. It should be removed.

File: src/ConsoleTableWriter.php [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        $this->table->setHeaders(array_keys($this->firstItem));
+        $this->table->render();
+
+        $this->firstItem = null;
+    }
+
+    /**
+     * @return Table
+     */
+    public function getTable()
```
Comment (sagikazarmark) [inline src/ConsoleTableWriter.php]: This getter should be removed

File: src/ConsoleProgressWriter.php (hunk 1 above)
Comment (ddeboer) [inline src/ConsoleProgressWriter.php]: :+1:

File: src/ConsoleProgressWriter.php (hunk 2 above)
Comment (ddeboer) [inline src/ConsoleProgressWriter.php]: :+1:

File: src/ConsoleTableWriter.php (hunk 4 above)
Comment (ddeboer) [inline src/ConsoleTableWriter.php]: :+1:

**Comment by sagikazarmark** [comment]
Unused code removed

**Comment by ddeboer** [comment]
Thanks!

**Pull request merged**